### DISCUSSION TITLE
怎么做到的呢

### DISCUSSION THREAD

**Discussion by STATJIANG** [yu-steven/openit#206] [General]
无偿使用吗

**Comment by yu-steven** [comment]
> 无偿使用吗

纯自动化，永久无偿提供
你想提供帮助我也接受😊

**Reply by STATJIANG** [reply]
头一次见走http模式👍我之前用的是trojan

**Reply by yu-steven** [reply]
用的方便就好😏

**Reply by STATJIANG** [reply]
怎么捐赠一下，有没有渠道

**Reply by yu-steven** [reply]
白嫖本无节点，都是找来的
捐赠的话，你想以什么形式？
你可以购买机场订阅等节点相关的方式

**Reply by STATJIANG** [reply]
我有订阅节点。支持你一下，感谢你的无私奉献！谢谢

**Reply by yu-steven** [reply]
🙏如何联系？
Telegram可否？[@yusteven](https://t.me/yusteven)
邮箱可否？yu@openit.ml

**Reply by STATJIANG** [reply]
已回电报

**Reply by yu-steven** [reply]
👌 OK
